README.md

```diff
@@ -15,6 +15,16 @@ FortiAI is a context-aware GenAI security assistant that transforms how security
 
 Its unique feature, among others, is its ability to generate playbook blocks tailored to scenarios defined by you, giving you a head start in automating and enhancing your use cases. Whether you are a seasoned playbook developer or just starting out, FortiAI is your go-to partner for streamlined and efficient playbook design.
 
+## Voice Support Overview
+
+FortiAI now includes voice-to-natural-language functionality, enabling hands-free interaction. This feature allows users to give voice commands for tasks such as generating playbooks, filtering records, or blocking indicators, streamlining workflows and increasing accessibility. This feature has multiple benefits like the following:
```

Comment:
It's a **Tap-and-Hold** feature so saying "hands-free interaction" seems counterintuitive.

Reply:
let it be.. :)